libs/sdk-model/src/execution/measure/index.ts

```diff
@@ -203,6 +203,21 @@ export function isSimpleMeasure(obj: any): obj is IMeasure<IMeasureDefinition> {
     return isMeasure(obj) && isMeasureDefinition(obj.measure.definition);
 }
 
+/**
+ * Type guard for checking whether object is an adhoc measure.
+ *
```

Comment:
please expand the documentation so that it describes what is an adhoc measure

Reply:
Thanks, I've updated the PR.